### PR TITLE
feat(ktabs): introduce kui tokens [khcp-7727]

### PR DESCRIPTION
**Pull request by aanchalm01**
# Summary
Addresses:
[https://konghq.atlassian.net/browse/KHCP-7727](https://konghq.atlassian.net/browse/KHCP-7727)

## PR Checklist

* [ ] **Conventional Commits** all commits follow the conventional commit standards [outlined in the main README](https://github.com/Kong/kongponents#committing-changes).
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: src/components/KTabs/KTabs.vue [hunk 1]
```diff
@@ -102,13 +102,13 @@ watch(() => props.modelValue, (newTabHash) => {
 
 .k-tabs {
   ul {
-    border-bottom: 1px solid var(--KTabsBottomBorderColor, var(--grey-300, color(grey-300)));
+    border-bottom: var(--kui-border-width-10, $kui-border-width-10) solid var(--KTabsBottomBorderColor, var(--grey-300, var(--kui-color-border-neutral-weak, $kui-color-border-neutral-weak)));
     display: flex;
-    font-size: 18px;
-    line-height: 20px;
+    font-size: var(--kui-font-size-50, $kui-font-size-50);
+    line-height: var(--kui-line-height-30, $kui-line-height-30);
```
Comment (adamdehaven) [inline src/components/KTabs/KTabs.vue:108]: the same step value for line-height should be used as font size in most scenarios, so here, it should be `line-height-50`

File: src/components/KTabs/KTabs.vue [hunk 2]
```diff
@@ -153,8 +153,8 @@ watch(() => props.modelValue, (newTabHash) => {
       &:hover {
         .tab-link.has-panels,
         .tab-link:not(.has-panels) :deep(> *) {
-          border-bottom: 4px solid var(--KTabBottomBorderColor, var(--teal-300, color(teal-300)));
-          color: var(--KTabsActiveColor, var(--black-500, color(black-500)));
+          border-bottom: var(--kui-border-width-30, $kui-border-width-30) solid var(--KTabBottomBorderColor, var(--teal-300, var(--kui-color-text-decorative, $kui-color-text-decorative)));
```
Comment (adamdehaven) [inline src/components/KTabs/KTabs.vue:156]: issue: you cannot use color-text tokens in border properties

File: src/components/KTabs/KTabs.vue (hunk 1 above)
Comment (aanchalm01) [reply src/components/KTabs/KTabs.vue:108]: But what was the reason that previously the value for font-size/line-height was different everywhere and now we are using same values ?

Comment (adamdehaven) [reply src/components/KTabs/KTabs.vue:108]: The design token step values don't equal each other.

For example:

```
kui-font-size-50 = 18px
kui-line-height-50 = 28px
```

Comment (aanchalm01) [reply src/components/KTabs/KTabs.vue:108]: updated.

File: src/components/KTabs/KTabs.vue (hunk 2 above)
Comment (aanchalm01) [reply src/components/KTabs/KTabs.vue]: fixed.

File: src/components/KTabs/KTabs.vue [hunk 3]
```diff
@@ -126,18 +127,18 @@ watch(() => props.modelValue, (newTabHash) => {
...
 
       .tab-link.has-panels,
       .tab-link:not(.has-panels) :deep(> *) {
         display: inline-block;
-        padding: var(--spacing-md, spacing(md));
+        padding: var(--spacing-md, var(--kui-space-60, $kui-space-60));
       }
 
-      &:not(:first-of-type) { margin-left: var(--spacing-xs, spacing(xs)); }
+      &:not(:first-of-type) { margin-left: var(--spacing-xs, var(--kui-space-40, $kui-space-40)); }
```
Comment (portikM) [inline src/components/KTabs/KTabs.vue:139]: Can we drop this to next line please?

File: src/components/KTabs/KTabs.vue [hunk 4]
```diff
@@ -126,18 +127,18 @@ watch(() => props.modelValue, (newTabHash) => {
...
 
-      &:not(:last-of-type) { margin-right: var(--spacing-xs, spacing(xs)); }
+      &:not(:last-of-type) { margin-right: var(--spacing-xs, var(--kui-space-40, $kui-space-40)); }
```
Comment (portikM) [inline src/components/KTabs/KTabs.vue:141]: Same here

File: src/components/KTabs/KTabs.vue (hunk 3 above)
Comment (aanchalm01) [reply src/components/KTabs/KTabs.vue:139]: Done

File: src/components/KTabs/KTabs.vue (hunk 4 above)
Comment (aanchalm01) [reply src/components/KTabs/KTabs.vue]: Done

**Review by portikM — DISMISSED** [review]
LGTM 🚀

File: docs/components/tabs.md [hunk 5]
```diff
@@ -396,6 +396,12 @@ export default {
 }
 </script>
 
+<style lang="scss" scoped>
+.horizontal-spacing {
```
Comment (DariaYeremina) [inline docs/components/tabs.md:400]: It looks like a utility class.
suggestion: rename it to `tab-1 `

Comment (aanchalm01) [reply docs/components/tabs.md:400]: I have been renaming them like this in md files. Will leave these ones as is.

**Review by jillztom — DISMISSED** [review]

**Review by portikM — APPROVED** [review]

**Review by jillztom — APPROVED** [review]

**Pull request merged**